webhooks-extension/docs/GettingStarted.md

```diff
@@ -103,14 +103,20 @@ Finally, select the Docker registry that any built images should be pushed to. T
 
 ## Putting it all together: test it's working
 
-Once a webhook is set up, a `git push` or merged Pull Request to a branch in the monitored repository should trigger the creation of a correct PipelineRun. This PipelineRun will show up in the Tekton dashboard as usual. As of August 2019, a successful webhook will trigger pod changes in the target namespace of the form shown below. 
+Once a webhook is set up, a `git push` or creation of a pull request to the monitored repository should trigger the creation of the correct PipelineRun. This PipelineRun will show up in the Tekton dashboard as usual. As of August 2019, a successful webhook will trigger pods to be creted as follows: 
+
+- Firstly, webhook receipt will drive a pod whose name will be of the format `tekton-xwxm5-rp6s6-gldvf-deployment-6674f7fdbb-pdgmw` created. 
 
-- Firstly, webhook receipt will drive a pod whose name corresponds to the 'Name' field of the webhook. E.g. for a Name of `buildah-hook` we might see a pod `buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-pdgmw` created. 
 - Next this pod will trigger the webhooks extension 'sink': a pod of the form `webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k` will execute. 
-- The 'sink' will then create the expected PipelineRun. The PipelineRun will spawn pods for each of its Tasks, so for example we might see a pod `buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e` created to run the `build-simple` Task. 
+
+- The 'sink' will then create the expected PipelineRun. The PipelineRun will spawn pods for each of its Tasks, so for example we might see a pod `buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e` created to run the `build-simple` Task.
+
+- If the webhook's event type is a pull request, an additional pod will be seen for the monitoring task.  This pod will monitor the PipelineRun and update the pull request with status.  The default monitoring task pod will be named similar to `pr-monitor-15657005244rsz8-pod-f5b42a`.  For more on monitoring see [here](Monitoring.md)
```

Comment:
Extra space before`For more on monitoring`

Reply:
and before "This pod" ... fixed